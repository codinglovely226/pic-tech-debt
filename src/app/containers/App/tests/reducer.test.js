import globalReducer from '../reducer';

describe('globalReducer', () => {
  it('should return the initial state', () => {
    expect(globalReducer(undefined, {})).toMatchSnapshot();
  });
});
