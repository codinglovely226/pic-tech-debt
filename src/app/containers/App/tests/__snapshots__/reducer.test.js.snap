exports[`globalReducer should return the initial state 1`] = `
Object {
  "cases": Object {},
  "incidents": Object {},
  "siteConfig": Object {
    "_id": "",
    "_rev": "",
    "siteId": "",
    "templates": Object {},
    "type": "siteConfig",
  },
  "vars": Object {
    "activeAgentTab": 0,
    "activeTab": 0,
    "agentView": Object {
      "attachments": Array [],
      "content": "",
      "show": false,
    },
    "agentViewCentreName": "",
    "applicationKeys": Object {},
    "callRecordId": null,
    "caseListSearch": Object {
      "callerLocation": "",
      "callerOrganisation": "",
      "centreName": "",
      "filter": "open",
      "name": "",
      "naturalId": "",
      "postcode": "",
      "sex": "",
      "time": "",
    },
    "currentRecordId": null,
    "currentTemplateId": null,
    "currentlySending": false,
    "currentlySyncing": false,
    "editingTemplateId": null,
    "editingTemplateMacro": Object {},
    "error": "",
    "formState": Object {
      "password": "",
      "username": "",
    },
    "incidentModal": Object {
      "caseId": "",
      "name": "",
      "selected": "",
      "show": false,
    },
    "investigationsViewRequest": Object {
      "content": "",
    },
    "investigationsViewResult": Object {
      "attachments": Array [],
      "content": "",
      "description": "",
      "requestIx": "",
      "show": false,
    },
    "jwtClaims": Object {},
    "loggedIn": false,
    "macro": "",
    "newCallModal": Object {
      "callerLocation": "",
      "callerOrganisation": "",
      "centreName": "",
      "filter": "all",
      "name": "",
      "naturalId": "",
      "postcode": "",
      "selected": null,
      "sex": "",
      "show": false,
      "time": "",
    },
    "quickView": Object {
      "category": "",
      "centreAgents": Array [],
      "dosage": Object {
        "quantity": "",
        "units": "",
        "unknown": false,
      },
      "eupcs": "",
      "exposure": Object {
        "duration": "",
        "durationUnits": "",
        "durationUnknown": false,
        "route": Array [],
        "timeSince": "",
        "timeSinceUnits": "",
        "timestamp": "",
        "type": "",
      },
      "id": "",
      "intendedUseGiven": "",
      "isPatientsOwnMedication": false,
      "name": "",
      "type": Array [],
    },
    "referralsView": Object {
      "referee": "",
      "refereeSite": "",
      "specialty": "",
    },
    "searchResults": Object {
      "actionBar": Array [],
      "caseList": Array [],
      "newRecord": Array [],
    },
    "swUpdated": false,
    "toxicologySource": "POISINDEX",
    "treatmentsView": Object {
      "content": "",
      "disposition": "",
      "spi": Object {
        "id": "",
        "job": "",
        "name": "",
        "org": "",
        "roles": Array [],
        "siteId": "",
      },
      "treatmentList": Array [],
    },
    "uiMode": "default",
    "views": Object {},
  },
}
`;
